ethpm/deployments.py

```diff
@@ -0,0 +1,5 @@
+class Deployments:
+
+    def __init__(self, deployment_data, w3):
+        self.deployment_data = deployment_data
```

Comment:
FYI, in the next pull request, we'll need to pass in the `contract_types` in some format.  I have some ideas on that that I'll post elsewhere.